contracts/HubPool.sol

```diff
@@ -26,13 +31,40 @@ contract HubPool is Testable, Lockable, MultiCaller, Ownable {
         bool isEnabled;
     }
 
-    WETH9Like public l1Weth;
+    enum RefundRequestStatus {
+        Pending,
+        Finalized,
+        Disputed
+    }
+
+    struct RelayerRefundRequest {
+        bytes32 poolRebalanceProof;
+        bytes32 destinationDistributionProof;
+        RefundRequestStatus status;
+    }
+
+    RelayerRefundRequest[] public relayerRefundRequests;
 
     // Whitelist of origin token to destination token routings to be used by off-chain agents.
     mapping(address => mapping(uint256 => address)) public whitelistedRoutes;
 
     mapping(address => LPToken) public lpTokens; // Mapping of L1TokenAddress to the associated LPToken.
 
+    // Address of L1Weth. Enable LPs to deposit/receive ETH, if they choose, when adding/removing liquidity.
+    WETH9Like public l1Weth;
```

Comment:
in general do we prefer storing contracts as addresses or as interfaces? I stored `weth` as an address in the `SpokePool` for example but we should just standardize

Reply:
ye, good point. my main question which should inform this decision is is this more expensive to store? is this more expensive to use?

my main reason for doing it this way is it means you dont need to do any type casting when using the interface which makes things a bit easier.